test/unit/BLSSignatureCheckerUnit.t.sol

```diff
@@ -9,28 +9,53 @@ contract BLSSignatureCheckerUnitTests is BLSMockAVSDeployer {
 
     BLSSignatureChecker blsSignatureChecker;
 
+    event StaleStakesForbiddenUpdate(bool value);   
+
     function setUp() virtual public {
         _setUpBLSMockAVSDeployer();
 
         blsSignatureChecker = new BLSSignatureChecker(registryCoordinator);
     }
 
+    function test_setStaleStakesForbidden_revert_notRegCoordOwner() public {
+        cheats.expectRevert("BLSSignatureChecker.onlyCoordinatorOwner: caller is not the owner of the registryCoordinator");
+        blsSignatureChecker.setStaleStakesForbidden(true);
+    }
+
+    function test_setStaleStakesForbidden() public {
+        testFuzz_setStaleStakesForbidden(false);
+        testFuzz_setStaleStakesForbidden(true);
+    }
+
+    function testFuzz_setStaleStakesForbidden(bool newState) public {
+        cheats.expectEmit(true, true, true, true, address(blsSignatureChecker));
+        emit StaleStakesForbiddenUpdate(newState);
+        cheats.prank(registryCoordinatorOwner);
+        blsSignatureChecker.setStaleStakesForbidden(newState);
+        assertEq(blsSignatureChecker.staleStakesForbidden(), newState, "state not set correctly");
+    }
+
     // this test checks that a valid signature from maxOperatorsToRegister with a random number of nonsigners is checked
     // correctly on the BLSSignatureChecker contract when all operators are only regsitered for a single quorum and
     // the signature is only checked for stakes on that quorum
-    function testBLSSignatureChecker_SingleQuorum_Valid(uint256 pseudoRandomNumber) public { 
+    function test_checkSignatures_SingleQuorum(uint256 pseudoRandomNumber) public { 
```

Comment:
This should probably be `testFuzz`, to differentiate from the function below with the same name